scheds/rust/scx_layered/src/stats.rs

```diff
@@ -301,12 +304,13 @@ impl LayerStats {
 
         writeln!(
             w,
-            "  {:<width$}  open_idle={} mig={} xnuma_mig={} xllc_mig={} affn_viol={}",
+            "  {:<width$}  open_idle={} mig={} xnuma_mig={} xllc_mig/skip={}/{} affn_viol={}",
```

Comment:
the `skip` might get confusing down the road, but I think it's fine for now as long as numa/llc related states are grouped together.